src/Sylius/Bundle/ReviewBundle/Resources/config/state-machine.yml.dist

```diff
@@ -0,0 +1,23 @@
+winzou_state_machine:
```

Comment:
It's exemplary file with state machine configuration, cause it should be configured with "reviewable" class name (as it is in ``CoreBundle`` with ``Product`` class).